src/Backend/Backend.ts

```diff
@@ -41,11 +44,19 @@ function backendRegistrationApi() {
       if (compiler) {
         gToolchainEnvMap[backend.name()] = new ToolchainEnv(compiler);
       }
+      const executor = backend.executor();
+      if (executor) {
+        globalExecutorArray.push(executor);
+      }
       Logger.info(logTag, 'Backend', backendName, 'was registered into ONE-vscode.');
+    },
+    registerExecutor(executor: Executor) {
```

Comment:
Q. Why is executor registered aside from Backend..? I thought it would included in backend.

Reply:
I thought there could be a Executor which is not related to Backend, so I make a room for this.

Reply:
but, if `Backend` could get multiple `Executor`, I think we don't keep this, as every `Executor` have each related `Backend`.

Reply:
I think #1076 change will remove this api as multiple executor could registerd on.